AWSutils.sh

```diff
@@ -354,12 +354,79 @@ function InstallCfnBootstrap {
     chroot "${CHROOTMNT}" alternatives --verbose --install /opt/aws/bin/cfn-hup cfn-hup /usr/local/bin/cfn-hup 1 --initscript cfn-hup || \
```

Comment:
Might as well remove the initscript option here, since that's what is creating the sysv script...

```suggestion
    chroot "${CHROOTMNT}" alternatives --verbose --install /opt/aws/bin/cfn-hup cfn-hup /usr/local/bin/cfn-hup 1 || \
```